frontend/src/modules/EditZing/Components/Helpers.tsx

```diff
@@ -47,3 +121,99 @@ export async function saveSwapStudent(
       }
     )
 }
+
+// makes a div that contains a Gender: Female type beat thing for student grid
+export function makeItem(svg: JSX.Element, text: string) {
+  // case for where the text is a graduation year, since those fluctuate year by year, we don't want to access the json
+  if (String(Number(text)) === text) {
+    return (
+      <div>
+        {svg} {text}
+      </div>
+    )
+  }
+
+  return (
+    <div>
+      {svg} {text}
+    </div>
+  )
+}
+
+export function getChipIcon(category: string): JSX.Element {
+  let svg = otherSvg(14)
+  switch (category) {
+    case 'graduation':
+      svg = yearSvg(14)
+      break
```

Comment:
If you want to save some lines the switch statement could be put in a little function and use `return` instead of assigning to svg and breaking. Then could actually use the default switch case lol